README.md

```diff
@@ -5,6 +5,9 @@ A time based / event series interactive visualization using d3.js. Use drag and
 
 ![EventDrops example](https://cloud.githubusercontent.com/assets/688373/18343222/c0a897b2-75b2-11e6-96df-e72e4b02335a.gif)
 
+
+Event Drops will support d3 version 4.x 
```

Comment:
@osadan: can you update this last paragraph manually? It should be done automatically via @sedy-bot normally (see https://marmelab.com/blog/2017/02/28/sedy-the-serverless-github-bot-which-fix-typos-for-you.html for more details), but apparently, it is offended. ;)

We should then be ready to merge.